library/src/containers/Error/styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
```

Comment:
Please change this styles like a other components. We decide to add for developers possibility to easy change styles by passing theme to root AsyncApi component.